packages/embed/src/utils/set-max-height.ts

```diff
@@ -0,0 +1,11 @@
+import { isMobile } from './mobile'
+
+export const setMaxHeight = (element: HTMLElement) => {
+  const mediaQuery = window.matchMedia('(max-width: 480px)')
```

Comment:
This seems a bit odd. Do we set height for screens under certain width? What if the device is rotated and has width > 480px?

Reply:
I added this media query because I see that on mobile, only below 480 we use the full height, if the popup or popover are above 480px they are not shown as full page. But yeah maybe could be removed and we always apply the maxHeight always? (In share page I solved this using styled-comp)

Reply:
Maybe we can get rid of the MQ at all and just leave the maxHeight? WDYT

Reply:
@mathio  @maxprilutskiy we can use css for this one, there is a `fill-available` intrinsic value that we can use for the max-height, it works on webkit and with the prefix will work also on Firefox. We do have few tickets related to this issue so the CSS solution might be better than using JS